packages/server/lambdas/alfresco/list-files.ts

```diff
@@ -71,18 +71,11 @@ const searchByTermWithParent = async (
 
 export const getFolder = async (uid: string, nodeId: string) => {
   try {
-    const alfrescoCoreAPIUrl = `${getAlfrescoUrlBase()}/alfresco/versions/1`;
-    const url = `${alfrescoCoreAPIUrl}/nodes/${nodeId}?where=(isFolder=true)&include=path`;
+    const url = `/alfresco/versions/1/nodes/${nodeId}?where=(isFolder=true)&include=path`;
     const options = await getAlfrescoOptions(uid, { 'Content-Type': 'application/json;charset=UTF-8' });
-    const response = await axios.get(url, options);
+    const response = await alfrescoAxios.get(url, options);
     return response.data;
   } catch (error: any) {
-    if (error instanceof AxiosError) {
-      // In case nodeId doesn't exist, Alfresco throws 404
-      if (error.response?.status === 404) {
```

Comment:
Referring to this in my other comment.

Reply:
@NinaDang97 do you remember about this? If this 404 check should be general or only in this list-files function?

Reply:
Only in list-file, in case no folder id is found then return empty result.

Reply:
Okay, I will remove it from the interceptor and use only in list-file 👍

Reply:
It's now handled it the interceptor for this function